src/library/components/GoogleLogin.jsx

```diff
@@ -0,0 +1,50 @@
+import { styled } from "@mui/material";
+import React from "react";
+// , { useState }
+import { GoogleLogin } from "react-google-login";
+import Colors from "../styleHelpers/customTheme";
+
+const CustomGoogleLogin = styled(GoogleLogin)`
+  width: 25vw;
+  height: auto;
+  margin: 10px;
+  box-shadow: none !important;
+  border: 1px solid ${Colors.primary}!important;
+  color: ${Colors.primary}!important;
+  border-radius: 14px !important;
+  justify-content: center; ;
+`;
+
+const GLogin = ({ children }) => {
+  const clientId =
+    "737423385254-lolcijd6n4834nn94heseis2gptsashc.apps.googleusercontent.com";
```

Comment:
Such sensitive keys should be loaded from env files.

Reply:
Yes, I will